submissions/7SergSmith7/html-movie-seat-booking/index.html

```diff
@@ -0,0 +1,791 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link
+      rel="stylesheet"
+      href="https://unpkg.com/modern-css-reset/dist/reset.min.css"
+    />
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" type="text/css" href="styles.css" />
+    <title>Movie Seat Booking</title>
+  </head>
+  <body>
+    <header>
+      <h1 class="header__title">Movie Seat Booking</h1>
```

Comment:
If you use BEM methodology. BEM element can't be outside BEM block. In your case, you use `.header_title` as BEM element without BEM block `.header`.
Also, no need to inherit all parent's hierarchy in class name.

Reply:
done